lib/workers/repository/onboarding/branch/create.ts

```diff
@@ -12,18 +12,23 @@ export function createOnboardingBranch(
   logger.debug('createOnboardingBranch()');
   const contents = getOnboardingConfig(config);
   logger.debug('Creating onboarding branch');
-  let commitMessage;
-  // istanbul ignore if
-  if (config.semanticCommits) {
-    commitMessage = config.semanticCommitType;
+
+  let prefix: string;
+  if (config.commitMessagePrefix) {
+    prefix = config.commitMessagePrefix;
```

Comment:
Treatment of commitMessagePrefix in onboarding should be consistent with its treatment in regular commit messages: https://github.com/renovatebot/renovate/blob/a8f27e0bf384d7298a655b44ba0a08e2e8839338/lib/workers/repository/updates/generate.ts#L189

i.e. the `:` is implicitly added if not present.

Reply:
Thanks for the code reference. I missed that when reviewing the repo. I'll make the update to keep it consistent.